keyboards/melgeek/mj64/mj64.c

```diff
@@ -0,0 +1,225 @@
+    {0, CS30_SW1, CS29_SW1, CS28_SW1}, /* RGB64 */
+    {0, CS27_SW1, CS26_SW1, CS25_SW1}, /* RGB65 */
+    {0, CS30_SW5, CS29_SW5, CS28_SW5}, /* RGB66 */
+};
+
+led_config_t g_led_config = {
+	{
+		/* C0     C1         C2        C3        C4        C5        C6        C7        C8          C9        C10         C11        C12        C13  */
+		{  0,      1,         2,        3,        4,        5,        6,        7,        8,          9,        10,         11,       12,        13     }, /* R0 */
+		{ 27,     26,        25,       24,       23,       22,       21,       20,       19,         18,        17,         16,       15,        14     }, /* R1 */
+		{ 28,     29,        30,       31,       32,       33,       34,       35,       36,         37,        38,         39,       40,        NO_LED }, /* R2 */
+		{ 54,     53,        52,       51,       50,       49,       48,       47,       46,         45,        44,         43,       42,        41     }, /* R3 */
+		{ 62,     61,        60,       NO_LED,   64,       63,       65,       NO_LED,   NO_LED,     59,        58,         57,       56,        55     }, /* R4 */
+    }, {
+        {0, 0},   {16, 0},   {32, 0},  {48, 0},  {64, 0},  {80, 0},  {96, 0},  {112, 0},  {128, 0},  {144, 0},  {160, 0},  {176, 0},  {192, 0},  {208, 0},
+		{0, 16},  {16, 16},  {32, 16}, {48, 16}, {64, 16}, {80, 16}, {96, 16}, {112, 16}, {128, 16}, {144, 16}, {160, 16}, {176, 16}, {192, 16}, {208, 16},
+		{0, 32},  {16, 32},  {32, 32}, {48, 32}, {64, 32}, {80, 32}, {96, 32}, {112, 32}, {128, 32}, {144, 32}, {160, 32}, {176, 32}, {192, 32},
+		{0, 48},  {16, 48},  {32, 48}, {48, 48}, {64, 48}, {80, 48}, {96, 48}, {112, 48}, {128, 48}, {144, 48}, {160, 48}, {176, 48}, {192, 48}, {208, 48},
+		{0, 64},  {16, 64},  {32, 64},           {48, 64}, {64, 64}, {80, 64},                       {96, 64},  {112, 64}, {128, 64}, {144, 64}, {160, 64},
+	}, {
+	       4,      4,         4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,        4, 
+		   4,      4,         4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,        4,
+		   4,      4,         4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,
+		   4,      4,         4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,        4,
+		   4,      4,         4,                  4,        4,        4,                              4,         4,          4,        4,        4,
+	}
+};
+
+#endif
+
+
+void suspend_power_down_kb(void) {
+    rgb_matrix_set_suspend_state(true);
+    suspend_power_down_user();
+}
+
+void suspend_wakeup_init_kb(void) {
+    rgb_matrix_set_suspend_state(false);
+    suspend_wakeup_init_user();
+}
+
+void keyboard_pre_init_kb(void) {
+    setPinOutput(A2);
+    writePinHigh(A2);
+    keyboard_pre_init_user();
+}
+
+#endif
```

Comment:
This should be split up to avoid the ifedf, and moved to a `mj64/<revision>/<revision>.c`